test/e2e/query_test.go

```diff
@@ -102,6 +104,155 @@ func sortResults(res model.Vector) {
 	})
 }
 
+func BenchmarkQueryRangeInterval(b *testing.B) {
```

Comment:
Will this benchmark run together with our CI pipeline? I am worried about it slowing down our CI process.
Can we add a skip and then we can run it locally if needed? WDYT

Reply:
No, benchmarks don't run in our CI pipeline :smile: perhaps they could in the future, it could be a LFX/GSoC project